pkg/kn/commands/types.go

```diff
@@ -43,19 +47,31 @@ func (c *KnParams) Initialize() {
 	}
 }
 
+// CurrentNamespace returns current namespace
```

Comment:
Maybe something like
// Return the current namespace which is either provided as option or picked up from kubeconfig